google/cloud/spanner/sql_statement_test.cc

```diff
@@ -105,6 +110,45 @@ TEST(SqlStatementTest, Equality) {
   EXPECT_NE(stmt1, stmt4);
 }
 
+TEST(SqlStatementTest, ToProtoStatementOnly) {
+  SqlStatement stmt("select * from foo");
+  internal::SqlStatementProto expected;
+  ASSERT_TRUE(TextFormat::ParseFromString(R"""(sql: "select * from foo")""",
+                                          &expected));
+  EXPECT_THAT(internal::ToProto(std::move(stmt)), IsProtoEqual(expected));
+}
+
+TEST(SqlStatementTest, ToProtoWithParams) {
+  SqlStatement::ParamType params = {
+      {"last", Value("Blues")},
+      {"first", Value("Elwood")},
+      {"destroyed_cars", Value(std::int64_t{103})},
+  };
+
+  const char* sql =
```

Comment:
Nit: since you don't actually care about the type of `sql`, meaning you don't care if it's an array or if it has been decayed to a pointer, consider just `auto sql = "....";`